content/ja/handbook/leadership/1-1/suggested-agenda-format.md

```
@@ -0,0 +1,91 @@
+translator: claude
+stale: false
+---
+
+このページでは推奨される1:1アジェンダフォーマットを説明しており、[Sid](/handbook/ceo)が1:1アジェンダをフォーマットする方法に基づいています。また、このフォーマットがデイリースタンドアップのアジェンダにどのように適用されるかも記載しています。
+[1:1](/handbook/leadership/1-1/)が初めての方や、GitLabの文化に合わせて以前の1:1の経験をどう活かすか不明な場合は、ここから始めるとよいでしょう。
+
+1:1はマネージャーと部下との間のプライベートミーティングですので、一貫性は効果性よりも二次的なものです。あなたとチームに適したフォーマットを見つけるようにしてください。
+
+## デイリースタンドアップのアジェンダ
+
+1:1アジェンダの「ハローウェイ」セクションは、デイリースタンドアップの推奨アジェンダフォーマットです。GitLabでは、デイリースタンドアップはイニシアチブを素早く整合させ加速するために存在します。限られた時間でスケジュールされています。主要なオープンアイテムの進捗を追跡するのに役立つため、[1:1ハローウェイ](#hallway)を使った修正版ミーティングアジェンダを使用します。週ごとに新しいアジェンダを作成しないため、週をまたいで重要な情報が失われるリスクが減ります。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)